_locales/en_US/messages.json

```diff
@@ -462,10 +462,10 @@
 		"message": "Display"
 	},
 	"options_showDKIMHeader": {
-		"message": "Show DKIM header"
+		"message": "Show DKIM header:"
```

Comment:
The idea behind leaving the `:` out here (and e.g. all the `options_error.<...>.treatAs`) was that is basically is a normall sentence when reading the label and selection together.
But I guess adding a `:` would be a little more consistent, so not against adding them.